routes/api.php

```diff
@@ -13,7 +14,11 @@
 | be assigned to the "api" middleware group. Make something great!
 |
 */
-
-Route::middleware('auth:sanctum')->get('/user', function (Request $request) {
-    return $request->user();
+Route::prefix('cities')->name('cities.')->group(function () {
```

Comment:
```suggestion
Route::controller(CityController::class)
    ->prefix('cities')
    ->name('cities.')
    ->group(function () {
```

Reply:
On a different note, I'd advise you to create a controller specific to APIs instead of using the same controller. Otherwise, we end up with a controller that handles both web and API requests. You could place it within an `Api` directory (of course, inside of the `Controllers` directory.